package.json

```diff
@@ -30,8 +32,26 @@
     "typescript": ">=3.0.0"
   },
   "dependencies": {
+    "@parcel/config-default": "^2.8.3",
     "@parcel/core": "^2.8.3",
     "@parcel/packager-ts": "^2.8.3",
-    "@parcel/transformer-typescript-types": "^2.8.3"
+    "@parcel/transformer-js": "^2.8.3",
+    "@parcel/transformer-react-refresh-wrap": "^2.8.3",
+    "@parcel/transformer-typescript-types": "^2.8.3",
+    "@preconstruct/cli": "^2.6.4"
+  },
+  "preconstruct": {
+    "packages": [
+      "packages/!(react-resizable-panels-website)"
+    ],
+    "globals": {
+      "react": "React"
+    },
```

Comment:
Did you mean to add this @Andarist? I can't see any UMD builds (and I'd personally recommend not adding them)

Reply:
No, I wasn't sure if UMDs were opt-in with Preconstruct so I included this and forgot to revisit it. Thanks for catching this! I removed it now.